_posts/2023-02-26-monero-GUI-0.18.2.0-released.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+title: "Monero GUI 0.18.2.0 'Fluorine Fermi' released"
+summary: Release containing redesigned wallet creation UI and support for the upcoming P2Pool network upgrade.
+tags: [releases]
+author: selsta
+image: /blog/assets/2022-07-19-monero-0.18.0.0-released/fluorine-fermi.png
+---
+
+<div align="center">
+  <img src="{{ page.image }}" width="350px">
+</div>
+
+# Overview
+
+This is the v0.18.2.0 release of the Monero GUI software. This release has a redesigned wallet creation UI and adds support for the upcoming P2Pool network upgrade.
```

Comment:
Recommended additional sentence just because you say "network upgrade", and it may be better to be overly clear to address any possible confusion:

> This is the v0.18.2.0 release of the Monero GUI software. This release has a redesigned wallet creation UI and adds support for the upcoming P2Pool network upgrade. This upgrade is not mandatory to use the Monero network unless you use P2Pool, but it is recommended.